app/src/main/res/layout/activity_schedule_response.xml

```diff
@@ -140,32 +140,45 @@
         android:background="@color/colorAccent"
         android:visibility="invisible" />
 
-    <LinearLayout
+    <TextView
+        android:id="@+id/text_view_schedule_response_confirmation"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
         android:layout_below="@+id/line_below_response"
-        android:gravity="center"
-        android:layout_margin="20dp"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginEnd="20dp"
+        android:text="@string/confirmation_of_schedule_response"
+        android:textSize="18dp" />
```

Comment:
should be sp instead of dp since this is a textSize.

Reply:
ok, done